packages/uma-auth-client/src/components/UmaConnectButton.tsx

```diff
@@ -1,26 +1,50 @@
-import { ThemeProvider } from "@emotion/react";
-import { Button } from "@lightsparkdev/ui/components";
-import { themes } from "@lightsparkdev/ui/styles/themes";
-import { GlobalStyles } from "src/GlobalStyles";
+import { css } from "@emotion/css";
+import styled from "@emotion/styled";
+import { Icon, UnstyledButton } from "@lightsparkdev/ui/components";
+import defineWebComponent from "src/utils/defineWebComponent";
 
-export const UmaConnectButton = () => {
+export const TAG_NAME = "uma-connect-button";
+
+const UmaConnectButton = () => {
   const clientId = "1";
-  const redirectUri = "http://localhost:3000";
+  const redirectUri = "http://localhost:3001";
   const responseType = "code";
   const codeChallenge = "1234";
   const codeChallengeMethod = "S256";
```

Comment:
Most of these will just be props IRL, right?

Reply:
right